models/_sector/tokens/tokens_transfers.sql

```diff
@@ -0,0 +1,75 @@
+{{config(
+    schema = 'tokens',
+    alias = 'transfers',
+    partition_by = ['token_standard', 'block_date', 'blockchain'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+    unique_key = ['unique_key', 'blockchain'],
+    post_hook='{{ expose_spells(\'["arbitrum", "avalanche_c", "base", "bnb", "celo", "ethereum", "fantom", "gnosis", "optimism", "polygon", "zksync", "zora"]\',
+                                "sector",
+                                "tokens",
+                                \'["aalan3", "jeff-dude"]\') }}'
+)
+}}
+
+{% 
+    set blockchains = [
```

Comment:
we should refer to the actual models here it makes jumping to the model easier.

Reply:
can you clarify on "jumping to model easier"?

Reply:
in vs code for example you can command click the model and go directly to the definition, it's quite nice.

Reply:
TIL 🤯 

i can go ahead and make that change. i like the look of this current approach, but can see the benefit of changing outweighing this